compiler-cli/src/config.rs

```diff
@@ -12,6 +13,35 @@ pub fn root_config() -> Result<PackageConfig, Error> {
     read(paths.root_config())
 }
 
+/// Get the config for a dependency module. Return the config for the current
+/// project if a dependency doesn't have a config file.
+pub fn module_config(mod_path: &str, project_paths: &ProjectPaths) -> Result<PackageConfig, Error> {
+    let package_path = fs::read_dir(project_paths.build_packages_directory()).map_or(None, |x| {
+        {
+            x.filter_map(|file| match file {
+                Ok(file) => Some(file),
+                Err(_) => None,
+            })
+            .find(|file| {
+                file.path()
+                    .join("src")
+                    .join(mod_path.to_string() + ".gleam")
```

Comment:
This will fail on Windows! We need to handle their paths too.

Reply:
Does this work? `.join(PathBuf::from(mod_path.to_string() + ".gleam"))` Been reading the PathBuf docs for a while an I can not tell if im doing this right.

Reply:
I changed the method to do this.